integrations-and-supported-tools/evidently/notebooks/Neptune_Evidently.ipynb

```diff
@@ -0,0 +1,565 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from neptune.utils import stringify_unsupported\n",
+    "\n",
+    "run[\"data_stability\"] = stringify_unsupported(data_stability.as_dict())\n",
+    "run[\"data_drift\"] = stringify_unsupported(data_drift_report.as_dict())"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Stop logging\n",
+    "\n",
+    "Once you are done logging, stop tracking the run."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "run.stop()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Analyze logged reports in the Neptune app\n",
+    "\n",
+    "Explore the run (reports, dictionaries) in the Neptune app, or check this [example dashboard](https://app.neptune.ai/o/common/org/evidently-support/runs/details?viewId=standard-view&detailsTab=dashboard&dashboardId=9917f940-757a-424d-879e-7781a00bf0c3&shortId=EV-7&type=run)."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Log production data drift\n",
+    "This section shows how you can use Evidently to evaluate production data drift and log the results to Neptune"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Load sample dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "! curl https://archive.ics.uci.edu/ml/machine-learning-databases/00275/Bike-Sharing-Dataset.zip --create-dirs -o data/Bike-Sharing-Dataset.zip\n",
+    "! unzip -o data/Bike-Sharing-Dataset.zip -d data\n",
+    "\n",
+    "bike_df = pd.read_csv(\"data/hour.csv\")\n",
+    "bike_df[\"datetime\"] = pd.to_datetime(bike_df[\"dteday\"])\n",
+    "bike_df[\"datetime\"] += pd.to_timedelta(bike_df.hr, unit=\"h\")\n",
+    "bike_df.set_index(\"datetime\", inplace=True)\n",
+    "bike_df = bike_df[\n",
+    "    [\n",
+    "        \"season\",\n",
+    "        \"holiday\",\n",
+    "        \"workingday\",\n",
+    "        \"weathersit\",\n",
+    "        \"temp\",\n",
+    "        \"atemp\",\n",
+    "        \"hum\",\n",
+    "        \"windspeed\",\n",
+    "        \"casual\",\n",
+    "        \"registered\",\n",
+    "        \"cnt\",\n",
+    "    ]\n",
+    "]\n",
+    "bike_df"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For demonstration purposes, we treat this data as the input data for a live model. To use with production models, the prediction logs should be available."
```

Comment:
> the prediction logs should be available

Maybe replace "should" with an unambiguous verb. If the logs have to be available &rarr; "must"

Reply:
This is copied verbatim from Evidently's documentation as this is concerned with how Evidently works and not within Neptune's scope